app/manage.py

```diff
@@ -1,42 +1,51 @@
 from flask_script import Manager
 from flask_migrate import Migrate, MigrateCommand
 from flask_script import Command
+from server import create_app, db
 import sys
 import os
 
 parent_dir = os.path.abspath(os.path.join(os.getcwd(), ".."))
 sys.path.append(parent_dir)
 sys.path.append(os.getcwd())
 
-from server import create_app, db
-from server.models.blockchain_address import BlockchainAddress
 
 class UpdateData(Command):
 
     def run(self):
```

Comment:
is this the seed? should we be adding this to the instructions for what to run in dev setup?

Reply:
yeah my plan was:
- move most of the actual logic of this into seed.py, and just call the function from here
- do the same with the other seed data.

Add the call to the readme.